src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -268,10 +268,12 @@ object TezosDatabaseOperations extends LazyLogging {
       copyAccountsToDelegateContracts(
         delegates.flatMap(_.content.values.flatMap(_.delegated_contracts)).toSet
       )
-    for {
+
+    (for {
       updated <- delegatesUpdateAction.map(_.sum)
       _ <- contractsUpdateAction
-    } yield updated
+    } yield updated).transactionally
```

Comment:
What's the reasoning for this being transactional now, as opposed to before?

Reply:
It was a bug, I effectively observed locally that the contracts's reference copy on db would fail because the delegate it was referencing (as a FK) wasn't yet transactionally visible to the second operation.